package.json

```diff
@@ -55,7 +55,7 @@
     "ldk": "^0.3.7",
     "lodash.debounce": "^4.0.8",
     "lottie-web": "^5.7.8",
-    "marina-provider": "^1.1.0",
+    "marina-provider": "github:louisinger/marina-provider#enrich-API",
```

Comment:
need merge of https://github.com/vulpemventures/marina-provider/pull/1

Reply:
You can pull from npm `v1.2.0`